### PR TITLE
chore: rekres with latest kres

### LINKED ISSUE #131: [feature] Add Version Number to Home page for Omni On-Prem
### Problem Description

For on-prem deployments, the Omni home page shows the currently assigned image tag as the ```Backend Version```.  If you are running with the "latest" image tag, the version simply shows as "latest"

For those of us who don't mind living on the bleeding edge, it would be nice to see the actual version number in use. 

### Solution

Instead of using the image tag for ```Backend Version```, show the actual backend version currently in use.

### Alternative Solutions

I've considered NOT using the "latest" image tag, but I like to live dangerously.

### Notes

_No response_

### PR DESCRIPTION
**Pull request by smira**
Fixes #131

### CODE REVIEW

File: client/.golangci.yml [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
 # options for analysis running
 run:
   timeout: 10m
   issues-exit-code: 1
   tests: true
-  build-tags: []
-  skip-dirs: []
-  skip-dirs-use-default: true
-  skip-files: []
+  build-tags: [ ]
```
Comment (utkuozdemir) [inline client/.golangci.yml:10]: is the change in formatting ok/intended?

**Review by utkuozdemir — APPROVED** [review]

File: client/.golangci.yml (hunk 1 above)
Comment (smira) [reply client/.golangci.yml:10]: not me, it's kres :)

Comment (utkuozdemir) [reply client/.golangci.yml:10]: I know, was wondering if it was expected from kres... But anyways, no big deal.

**Comment by DmitriyMV** [comment]
@smira @utkuozdemir  since we download and build `golangci-lint` ourselves, maybe it makes sense to just temporary bind to the Go pseudo-version of it? Like `v1.52.3-whatevergooutputsforthelatesthash`.

**Comment by smira** [comment]
but on a second thought, I could only bump in Omni

**Review by Unix4ever — APPROVED** [review]

**Pull request closed without merging**